package.json

```diff
@@ -32,7 +32,8 @@
   "dependencies": {
     "axios": "0.19.0",
     "commander": "2.17.1",
-    "debug": "3.1.0"
+    "debug": "3.1.0",
+    "http-status": "^1.4.2"
```

Comment:
Please remove caret like the other dependencies, we are locking versions for various reasons.

Reply:
caret removed